data.js

```diff
@@ -8,6 +8,14 @@
 // See examples below.
 
 module.exports = [
+  {
+    name: "Combo.js",
+    github: "combojs/combo-js",
+    tags: ["component", "framework", "ui", "templating"],
+    description: "A lightweight library for building user interfaces",
+    url: "https://combojs.com/",
```

Comment:
It seems it not support https, currently. Is it typo or you will get https soon?